src/object.d

```diff
@@ -3241,6 +3241,269 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+private template Floating1(T)
```

Comment:
Why the name? Never would guess that something called `Floating1` deals with comparison.